### DISCUSSION TITLE
Standard options: unit: SI prefix needed

### DISCUSSION THREAD

**Discussion by jelmd** [grafana/grafana#41229] [Dashboards, Panels & Visualizations]
For big (or small) values (counters) the only way to get a half way compact value shown on the Y-Axis, tooltips, legends, etc. is to use `Standard options | Unit | Misc | short`. But it has its problems. E.g. `Bil`: Beside that it is still using 3 letters it is ambiguous - the user looking at the chart cannot be certain, whether  it means 10^9 or 10^12. Similar `Mil`: Many germans (and probably other peoples) would wonder, whether it means `Millionen` or `Milliarden` ...

However, most users looking at charts know [SI prefixes]( https://www.nist.gov/pml/weights-and-measures/metric-si-prefixes) and consume such readings as is, i.e. w/o starting an explicit conversion process in its brain. Thus having a `Standard options | Unit | Misc | SI prefix` would make a lot of charts more human readable, unambiguously and many people happier :) .

**Comment by raymalt** [comment]
This would be very beneficial in one of our use cases, where different power measures are plotted alongside each other on the same y axis, e.g. W plotted together with VAr and VA.

**Comment by raymalt** [comment]
Issue created: https://github.com/grafana/grafana/issues/78022
PR opened: https://github.com/grafana/grafana/pull/78023

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!

**Comment by raymalt** [comment]
Just for the record, there is an approved PR (https://github.com/grafana/grafana/pull/79897) implementing the request discussed here.
